recipe/meta.yaml

```diff
@@ -53,7 +53,7 @@ requirements:
     - libtiff
     - libuuid  # [linux]
     - libwebp-base
-    - libxml2
+    - libxml2 2.10
     - lz4-c
     - openjpeg
     - openssl
```

Comment:
what about other packages pinning ?

Reply:
Since it's a rebuild I prefer keeping it as is for now.